src/demos/calcite-input.html

```diff
@@ -322,7 +322,8 @@ <h4>Readonly</h4>
           <calcite-label>
             Readonly with prefix and button
             <calcite-input readonly prefix-text="API Key" value="A123-S5DL-24FA">
-              <calcite-button slot="input-action" onclick="alert('copied')" icon-start="copy-to-clipboard"></calcite-button>
+              <calcite-button slot="input-action" onclick="alert('copied')" icon-start="copy-to-clipboard">
```

Comment:
Nitpick: whitespace change intentional?

Reply:
Formatted, I think the whole demo page needs a Prettier pass so I'll commit in next cleanup.